device/transport/mqtt/devdoc/mqtt_requirements.md

```diff
@@ -300,17 +260,17 @@ The `getTwinReceiver` method creates a `MqttTwinReceiver` object for the twin re
 
 **SRS_NODE_DEVICE_MQTT_16_053: [** `enableMethods` shall call its callback with an `Error` if subscribing to the topic fails. **]**
 
-### enableTwin
+### enableTwinDesiredPropertiesUpdates
 
-**SRS_NODE_DEVICE_MQTT_16_057: [** `enableTwin` shall connect the MQTT connection if it is disconnected. **]**
+**SRS_NODE_DEVICE_MQTT_16_057: [** `enableTwinDesiredPropertiesUpdates` shall connect the MQTT connection if it is disconnected. **]**
 
-**SRS_NODE_DEVICE_MQTT_16_058: [** `enableTwin` shall calls its callback with an `Error` object if it fails to connect. **]**
+**SRS_NODE_DEVICE_MQTT_16_058: [** `enableTwinDesiredPropertiesUpdates` shall calls its callback with an `Error` object if it fails to connect. **]**
 
-**SRS_NODE_DEVICE_MQTT_16_059: [** `enableTwin` shall subscribe to the MQTT topics for twins. **]**
+**SRS_NODE_DEVICE_MQTT_16_059: [** `enableTwinDesiredPropertiesUpdates` shall subscribe to the MQTT topics for twins. **]**
```

Comment:
>** `enableTwinDesiredPropertiesUpdates` shall subscribe to the MQTT topics for twins. **]** [](start = 32, length = 91)

This is inconsistent with your requirements above.  The new requirements above are about calling functions in other objects.  This one is about actually subscribing, which implies calling the base transport from here, which seems broken.

Reply:
true, i think it should just say "calls the enableTwinDEsiredPropertiesUpdate on the twin client"

---
In reply to: [170614894](https://github.com/Azure/azure-iot-sdk-node/pull/236#discussion_r170614894) [](ancestors = 170614894)